File: test/end-to-end/Classes/Statics/static functions.cs
//
using System;

class Utilities
{
    public static string GetLongest(string[] words)
    {
        string longest = "";

        foreach (string word in words)
        {
            if (word.Length > longest.Length)
            {
                longest = word;
            }
        }

        Utilities.Log(word);

        return longest;
    }

    public static void Log(string word)
    {
        Console.WriteLine("Logging: " + word);
    }
}
//

File: test/end-to-end/Classes/Statics/static functions.java
class Utilities {
    public static string getLongest(string[] words) {
        string longest = "";

        for (string word : words) {
            if (word.length() > longest.length()) {
                longest = word;
            }
        }

        Utilities.log(word);

        return longest;
    }

    public static void log(string word) {
        System.out.println("Logging: " + word);
    }
}
